### ISSUE TITLE
vault value update detect

### ISSUE THREAD

**Issue by marcus-sds** [Comcast/concourse-vault-resource#1]
Hi. Is it possible to detect vault value change and trigger task?

Thank you.

**Comment by shellfu** [comment]
Yes this is certainly possible. You would need to set that piece up in the pipeline itself based on this resource finding the update.

If you have any issues with this please feel free to elaborate on any troubles you're encountering.
